bazel/revisions.bzl

```diff
@@ -54,6 +54,7 @@ EMSCRIPTEN_TAGS = {
     "3.1.21": struct(
         hash = "a16a8bca2466eb144f7c93fa899c0272c8815dc3",
         sha_linux = "7045ddb3b37a2cc63cb1cf976019a6a3b7f8dbdc71254db0eee5b0452f94e9e7",
+        sha_linux_arm64 = "2852c8b108ec748d52d31dab3f4854bc6022df008019daff1c7e31ac00363b3f",
```

Comment:
`3.1.21` is the only linux arm64 binaries I could find the sha for. If there's a way to find/list shas for more I'd be happy to add them.

For this one:

https://github.com/emscripten-core/emsdk/blob/5b80c10e5660ac799580ded0d7c5e1a5f253c65e/emscripten-releases-tags.json#L27
```
$ curl -sSL https://storage.googleapis.com/webassembly/emscripten-releases-builds/linux/a16a8bca2466eb144f7c93fa899c0272c8815dc3/wasm-binaries-arm64.tbz2 | shasum -a 256
2852c8b108ec748d52d31dab3f4854bc6022df008019daff1c7e31ac00363b3f
```